lib/BootBot.js

```diff
@@ -58,13 +58,32 @@ class BootBot extends EventEmitter {
   }
 
   sendGenericTemplate(recipientId, elements, options) {
+    var image_aspect_ratio = options && options.image_aspect_ratio && options.image_aspect_ratio.toLowerCase();
+    image_aspect_ratio = (image_aspect_ratio !== 'horizontal' && image_aspect_ratio !== 'square') ? 'horizontal' : image_aspect_ratio;
+    options && delete options.image_aspect_ratio;
     const payload = {
       template_type: 'generic',
+      image_aspect_ratio,
       elements
     };
     return this.sendTemplate(recipientId, payload, options);
   }
 
+  sendListTemplate(recipientId, elements, buttons, options) {
+    var top_element_style = options && options.top_element_style && options.top_element_style.toLowerCase();
+    top_element_style = (top_element_style !== 'large' && top_element_style !== 'compact') ? 'large' : top_element_style;
+    options && delete options.top_element_style;
+    buttons.length = 1;
```

Comment:
The `length` property is a read-only property, I'm afraid this could cause issues in certain environments. A better way to force the array to just have one element would be to do: `buttons = [buttons[0]];`